.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```

Comment:
Why is anything related to Vagrant in this PR, we've got Docker and docker-compose already why do we need Vagrant?